graspologic/utils/utils.py

```diff
@@ -971,10 +975,16 @@ def remap_node_ids(
     if not isinstance(graph, nx.Graph):
         raise TypeError("graph must be of type nx.Graph")
 
+    if not nx.is_weighted(graph, weight=weight_attribute):
+        logger = logging.getLogger("graspologic.utils")
```

Comment:
I was thinking the warnings.warn module actually.

Reply:
Updated to use warnings module